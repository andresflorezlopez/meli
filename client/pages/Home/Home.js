import React from "react";
import MainLayout from "@layouts/MainLayout/MainLayout";

const Home = () => {
  return <MainLayout />;
};

export default Home;
