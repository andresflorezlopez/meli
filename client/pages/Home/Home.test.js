import React from "react";
import Home from "./Home";
import renderer from "react-test-renderer";
import LightTheme from "@config/themes/light";
import { ThemeProvider } from "styled-components";

describe("Home", () => {
  test("should render", () => {
    const component = renderer.create(
      <ThemeProvider theme={LightTheme}>
        <Home />
      </ThemeProvider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
