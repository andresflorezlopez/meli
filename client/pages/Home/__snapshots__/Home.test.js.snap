// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render 1`] = `
<div>
  <div
    className="sc-18pr4qs-0 eyHxCt"
  >
    <div
      className="sc-1os0qt1-0 ecarEL"
      direction="row"
    >
      <div
        className="sc-18pr4qs-1 jjdFVY"
      >
        <img
          alt="Mercado Libre"
          className="sc-18pr4qs-2 hvNeoX"
          onClick={[Function]}
          src="test-file-stub"
        />
      </div>
      <div
        className="sc-18pr4qs-3 ePKDzW"
      >
        <input
          className="sc-18pr4qs-4 gfnSjD"
          defaultValue=""
          onChange={[Function]}
          onKeyPress={[Function]}
          placeholder="Nunca dejes de buscar"
        />
        <div
          className="sc-18pr4qs-5 bUzMWw"
          onClick={[Function]}
        />
      </div>
    </div>
  </div>
</div>
`;
